presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergTableProperties.java

```diff
@@ -35,6 +37,7 @@ public class IcebergTableProperties
     public static final String PARTITIONING_PROPERTY = "partitioning";
     public static final String LOCATION_PROPERTY = "location";
     public static final String FORMAT_VERSION = "format_version";
+    public static final String CONCURRENCY_RETRIES = "commit_retries";
```

Comment:
Why name the constant `CONCURRENCY_RETRIES` when the property is `commit_retries`?

Reply:
Will change